src/Adapter/Attribute/AdminAttributeGeneratorControllerWrapper.php

```diff
@@ -43,7 +42,7 @@
 class AdminAttributeGeneratorControllerWrapper
```

Comment:
it feels weird that this class keeps being called `AdminAttributeGeneratorControllerWrapper` when `AdminAttributeGeneratorController` doesn't exist anymore...  and then to add core logic in this "wrapper

Reply:
It is since 1700.

I think this old code will be migrated when the attribute page will be migrated.